.github/workflows/publish.yml

```diff
@@ -0,0 +1,41 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Publish to PyPI
+
+on:
+  release:
+    types: [created]
```

Comment:
Hmmm, yes, looking at the docs that seems to make more sense. Good catch. I took the basic code from this example: https://github.com/actions/starter-workflows/blob/master/ci/python-publish.yml

I wonder why they are using `created` though, since I don't see many use cases where one would want to push a draft release to PyPI? https://developer.github.com/webhooks/event-payloads/#webhook-payload-object-34